### ISSUE TITLE
AutoLayout and custom font

### ISSUE THREAD

**Issue by MBLESSEN** [optonaut/ActiveLabel.swift#45]
This was mentioned in a another issue but my problem is specific to font.

When I use a custom font, PTSans, the text is cut off at the end. 

**this is what it should look like:**

<img width="354" alt="screen shot 2016-02-11 at 12 43 02 pm" src="https://cloud.githubusercontent.com/assets/10052663/12986495/319ff834-d0bd-11e5-8b31-cb2db6bbbb7d.png">

**And this is what it looks with with ActiveLabel**

<img width="359" alt="screen shot 2016-02-11 at 12 43 58 pm" src="https://cloud.githubusercontent.com/assets/10052663/12986487/2703313e-d0bd-11e5-9a6f-9cff359b8038.png">

**Notice the text and time stamp at the end is missing**

This is a label inside a tableview cell using Automatic dimensions for the height. the cell height changes based on the size of the label.

**Comment by MBLESSEN** [comment]
I am not using size that fits

**Comment by poolqf** [comment]
Hi @MBLESSEN, @kaandedeoglu told you about a possible solution:

> @MBLESSEN Autolayout support might also require `intrinsicContentSize` to be overridden. Can you please try if

>```swift
let labelHeight = label.sizeThatFits(CGSize(width: CGRectGetWidth(label.frame), height: CGFloat.max)).height

> label.frame.size.height = labelHeight
```

> gives the correct height.

I've recently merged #46, and I would like to know if any of those fixes are working for you

**Comment by skrew** [comment]
Works for me by adding in `ActiveLabel.swift`:
```
public override func intrinsicContentSize() -> CGSize {
        let labelHeight = self.sizeThatFits(CGSize(width: CGRectGetWidth(self.frame), height: CGFloat.max)).height
        return CGSize(width: self.frame.size.width, height: labelHeight + self.layoutMargins.top + self.layoutMargins.bottom)
    }
```
I don't know if this patch are correct but it's works for me.
